abstractions/ruby/microsoft_kiota_abstractions/lib/microsoft_kiota_abstractions/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module MicrosoftKiotaAbstractions
-  VERSION = "0.3.0"
+  VERSION = "0.4.0"
```

Comment:
I'm not sure why you bumped the versions in the ruby libraries since no changes were made? Did we forget to bump them and release last time?
You can fix that by running:

```shell
git reset HEAD~1
git checkout **/version.rb
git add *.cs *.md
git commit -m "your commit comment"
git push -f
```